.travis.yml

```diff
@@ -7,4 +7,4 @@ cache:
 install:
   - npm install
 script:
-  - npm test -- --coverage
\ No newline at end of file
+  - npm test
```

Comment:
how come you changed this line

Reply:
Added the coverage option to the test script so that we don't have to add it explicitly every time we run them locally